config/locales/rdr.en.yml

```diff
@@ -18,6 +18,7 @@ en:
     unexpected_multiple_results: "Unexpected multiple results for %{identifier}"
     user_not_found: "User %{user_key} could not be found"
     not_found: "%{target} could not be found"
+    work_files_stats: "%{count} files (%{size})"
```

Comment:
@coblej - could we change this to use the 'one/other' options in order to account for single files?
(see https://robots.thoughtbot.com/pluralizing-rails-i18n-translations)

Reply:
@mikedaul Thanks for the link.  Fixing this turned out to be much easier than I thought based on something else I had seen earlier.